src/components/SubHeader/EvergreenTaskSubHeader.tsx

```diff
@@ -0,0 +1,49 @@
+import { useQuery } from "@apollo/client";
+import { usePreferencesAnalytics } from "analytics";
+import Icon from "components/Icon";
+import { StyledLink } from "components/styles";
+import TaskStatusBadge from "components/TaskStatusBadge";
+import { getEvergreenTaskURL } from "constants/externalURLTemplates";
+import { GetTaskQuery, GetTaskQueryVariables } from "gql/generated/types";
+import { GET_TASK } from "gql/queries";
+
+interface Props {
+  taskID: string;
+  execution: number;
+}
+
+export const EvergreenTaskSubHeader: React.FC<Props> = ({
+  taskID,
+  execution,
+}) => {
+  const { sendEvent } = usePreferencesAnalytics();
+  const { data, loading } = useQuery<GetTaskQuery, GetTaskQueryVariables>(
+    GET_TASK,
+    {
+      variables: { taskId: taskID as string, execution: Number(execution) },
+    }
+  );
+
+  const { task } = data ?? {};
+  const taskLink = getEvergreenTaskURL(taskID, execution);
+
+  return (
+    <>
+      <Icon glyph="EvergreenLogo" size={24} />
+      {!loading && (
+        <>
+          <StyledLink
+            data-cy="spruce-link"
+            href={taskLink}
+            onClick={() => sendEvent({ name: "Opened Task Link" })}
+            target="_blank"
+            title="Open task in Spruce"
+          >
+            {task?.displayName || "Task"}
```

Comment:
Could we include a more complete breadcrumb?
The designs feature something like this
<img width="415" alt="image" src="https://user-images.githubusercontent.com/4605522/228319233-f5243ad5-3b9e-4044-a714-64fe0a16833f.png">

<img width="669" alt="image" src="https://user-images.githubusercontent.com/4605522/228319423-d81232a9-6c0a-415c-a9db-9d5424992840.png">

<img width="1272" alt="image" src="https://user-images.githubusercontent.com/4605522/228321094-a6775f5d-53c1-4590-bebd-be12553d6674.png">

Reply:
Fwiw I don’t feel that breadcrumbs make a ton of sense here. They’re useful for situating users in an information hierarchy but Parsley has no such hierarchy, Spruce does. Linking them directly to the task which then contains the breadcrumbs seems reasonable and less cluttered, but 🤷‍♀️